packages/disputer/test/Disputer.js

```diff
@@ -504,9 +495,7 @@ contract("Disputer.js", function (accounts) {
             // Note the check below has a bit of switching logic that is version specific to accommodate the change in withdrawal behaviour.
             assert.equal(
               (await financialContract.getLiquidations(sponsor1))[0].state,
-              contractVersion.contractVersion == "1.2.2"
```

Comment:
Looks like we can also remove all references to `this.isLegacyEmpVersion` in the source code! For example [here](https://github.com/UMAprotocol/protocol/blob/master/packages/disputer/src/disputer.js#L111)

Reply:
I want to leave this behavior for a separate PR and leave this one just for tests